go.mod

```diff
@@ -3,37 +3,26 @@ module github.com/openshift/console-operator
 go 1.16
 
 require (
-	github.com/Microsoft/go-winio v0.4.11 // indirect
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/containerd/continuity v0.0.0-20190827140505-75bee3e2ccb6 // indirect
-	github.com/coreos/etcd v3.3.15+incompatible // indirect
 	github.com/davecgh/go-spew v1.1.1
-	github.com/docker/go-connections v0.3.0 // indirect
-	github.com/docker/libnetwork v0.0.0-20190731215715-7f13a5c99f4b // indirect
-	github.com/fsouza/go-dockerclient v0.0.0-20171004212419-da3951ba2e9e // indirect
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-bindata/go-bindata v3.1.2+incompatible
 	github.com/go-test/deep v1.0.5
 	github.com/google/gofuzz v1.2.0 // indirect
-	github.com/googleapis/gnostic v0.5.5 // indirect
-	github.com/kubernetes-sigs/kube-storage-version-migrator v0.0.0-20191127225502-51849bc15f17 // indirect
-	github.com/morikuni/aec v1.0.0 // indirect
 	github.com/open-cluster-management/api v0.0.0-20210527013639-a6845f2ebcb1
-	github.com/opencontainers/runc v0.0.0-20191031171055-b133feaeeb2e // indirect
 	github.com/openshift/api v0.0.0-20220824134416-0f86a223d4bc
 	github.com/openshift/build-machinery-go v0.0.0-20211213093930-7e33a7eb4ce3
 	github.com/openshift/client-go v0.0.0-20220525160904-9e1acff93e4a
 	github.com/openshift/library-go v0.0.0-20220823205642-e2ef049de560
 	github.com/pkg/profile v1.4.0 // indirect
 	github.com/spf13/cobra v1.4.0
-	github.com/vishvananda/netlink v1.0.0 // indirect
-	github.com/vishvananda/netns v0.0.0-20191106174202-0a2b9b5464df // indirect
 	gopkg.in/yaml.v2 v2.4.0
 	k8s.io/api v0.24.0
 	k8s.io/apimachinery v0.24.0
-	k8s.io/apiserver v0.24.0 // indirect
 	k8s.io/client-go v0.24.0
 	k8s.io/component-base v0.24.0
 	k8s.io/klog/v2 v2.60.1
 	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
 )
+
+replace github.com/opensihft/api => github.com/debsmita1/api v0.0.0-20220907082345-66edd0033bf1
```

Comment:
I replaced this so that we can test the console-operator changes and the console changes together on a dev cluster.
I will remove this once the api gets merged